liota/dccs/iotcc.py

```diff
@@ -157,7 +159,15 @@ def on_response(msg, reg_resp_q):
                     self._store_reg_entity_attributes("Devices", entity_obj.name, self.reg_entity_id,
                                                       entity_obj.entity_type, None)
 
-            return RegisteredEntity(entity_obj, self, self.reg_entity_id)
+            _reg_entity_obj = RegisteredEntity(entity_obj, self, self.reg_entity_id)
+            if self.sys_properties is not None and self.sys_properties:
+                if ast.literal_eval(self.sys_properties):
+                    self.set_properties(_reg_entity_obj, ast.literal_eval(self.sys_properties))
+                    log.info(
+                        "System Properties defined for the resource {0}".format(entity_obj.name))
```

Comment:
Can you please add the properties as well in the log? It will be useful to debug issues.